libs/server/Resp/RespServerSession.cs

```diff
@@ -299,6 +312,9 @@ public override int TryConsumeMessages(byte* reqBuffer, int bytesReceived)
             }
             finally
             {
+                // reset the session's flag for AOF blocking to default value after processing all commands
+                waitForAofBlocking = false;
```

Comment:
Move this to https://github.com/microsoft/garnet/pull/511/files#diff-b42721ea9aeab99ac7f5177323136d172b8d0c441c03df3f3634f96aa1774ac6L907

Reply:
moved it back to conservative approach, after SendAndReset was discussed with you. MArking this as resolved.